common/tee.edl

```diff
@@ -15,6 +15,7 @@
 enclave
 {
     include "openenclave/bits/types.h"
+    include "openenclave/internal/switchless.h"
```

Comment:
If we're going to make this an edl-defined ecall then the function should be in a public header.

Reply:
Resolved.